src/nnet3/nnet-simple-component.h

```diff
@@ -309,6 +310,7 @@ class AffineComponent: public UpdatableComponent {
   Component *CollapseWithNext(const FixedAffineComponent &next) const;
```

Comment:
Convolution1dComponent might be deprecated, but ConvolutionComponent also has the same issue (see #308 ). It would be great if you rectify this problem.